pkg/buildah/diff.go

```diff
@@ -100,15 +102,16 @@ func changesToTable(out io.Writer, diffs []archive.Change) error {
 
 type patchOption struct {
 	createPatchFile bool
-	build           bool
-	save            bool
+	build           string
+	save            string
 }
 
 func (o *patchOption) RegisterFlags(fs *pflag.FlagSet) {
-	fs.BoolVar(&o.createPatchFile, "create-patch", false, "If enabled then create a named archive file contains changes that we interested")
-	fs.BoolVar(&o.build, "build", false, "Auto build a new image with the generated changes")
-	fs.BoolVar(&o.save, "save", false, "Auto save the built image into a oci-archive file")
-	bailOnError(markFlagsHidden(fs, "create-patch", "build", "save"), "")
+	fs.BoolVar(&o.createPatchFile, "create-patch", false, "if enabled then create a named archive file contains changes that we interested")
+	fs.StringVar(&o.build, "build", "", "tag name, diff will auto build a new image with the generated changes if flag specified"+
```

Comment:
Why tag name call it `build`? Provide an example about how to use those flags.

Reply:
> Why tag name call it `build`? Provide an example about how to use those flags.

What would you recommend from a user experience perspective?